backend-api/src/main/java/org/wcci/apimastery/Controllers/AlbumController.java

```diff
@@ -15,6 +16,7 @@
 public class AlbumController {
 
     private AlbumStorage albumStorage;
+    private SongStorage songStorage;
 
     public AlbumController(AlbumStorage albumStorage) {
         this.albumStorage = albumStorage;
```

Comment:
```suggestion
    public AlbumController(AlbumStorage albumStorage, SongStorage songStorage) {
        this.albumStorage = albumStorage;
        this.songStorage = songStorage;
```
Needs instantiated in constructor